### PR TITLE
Revisions of Landing-Page for Core-Protocol WG

### PR DESCRIPTION
**Pull request by AlexHentschel**
* first complete roadmap draft;
* included  links to the `Flow Core-Protocol Vision` and `Data Availability Vision`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,16 +24,183 @@ Evolution of Flow to the mature protocol. This includes:
...
+
+#### Dynamic Protocol State as a foundation for thread response capabilities, e.g. ban slashed node, revoke compromised keys, software upgrades
+
+This work is a necessary premise for the Flow network to autonomously defend itself against
+malicious nodes within the network. On a technical level, the Dynamic Protocol State is the foundation for
+countermeasures such as slashing misbehaving nodes or entirely revoking their authorization to participate.
+In a nutshell, the Dynamic Protocol State allows for updating various protocol parameters,
+such as permissions to participate, during the live operations of the Flow network.
+
+* Status: implementation completed (:point_right: [code on gitbub](https://github.com/onflow/flow-go/tree/feature/dynamic-protocol-state))
```
Comment (jordanschalm) [inline README.md:64]:
```suggestion
* Status: implementation completed (:point_right: [code on GitHub](https://github.com/onflow/flow-go/tree/feature/dynamic-protocol-state))
```

File: README.md [hunk 2]
```diff
@@ -19,16 +24,183 @@ Evolution of Flow to the mature protocol. This includes:
...
+As a result, the honest supermajority becomes resilient against a byzantine minority trying to consume network resources exploitatively.
+
+* Status: Advanced stage of development.
+
+  Currently, Flow can tolerate coordinated attacks of 5 malicious nodes, which will increase to 12 within the next few months.
+  For non-colluding malicious nodes, the networking layer can withstand attacks of a significantly higher number already.
+  Furthermore, we have detailed logic in place for node operators to blacklist other nodes as a manual fallback mechanism to
+  defend against attacks that are not yet automatically handled. Later this year, Flow will be able to withstand attacks (colluding and non-colluding)
+  of up to ⅓ of all nodes, which is the theoretical maximum for consensus under partially-synchronous networking conditions.
+* Completion date: uncertain but likely within 2024
```
Comment (jordanschalm) [inline README.md:106]:
```suggestion
* Target completion date: Late 2024
```
All completion dates have some uncertainty. I feel like stating the target date, as a "target" is the clearest way to communicate these dates.

File: README.md [hunk 3]
```diff
@@ -19,16 +24,183 @@ Evolution of Flow to the mature protocol. This includes:
...
+
+
+#### Path to Zero-downtime protocol upgrades
+
+In 2023, Flow had an uptime of 99.95%. In 2024, we want to further reduce the number of sporks for uninterrupted network availability.
+During earlier stages, the ability to take down the network for a software upgrade has greatly helped the development
+speed of Flow. Though, as the flow network matures, we need more sophisticated tools to deploy software upgrades to a running network
```
Comment (jordanschalm) [inline README.md:113]:
```suggestion
speed of Flow. Though, as the Flow network matures, we need more sophisticated tools to deploy software upgrades to a running network
```

File: README.md [hunk 4]
```diff
@@ -19,16 +24,183 @@ Evolution of Flow to the mature protocol. This includes:
...
+At the moment, the Flow network already has Height-Coordinated Updates [HCU] as a tool to deploy protocol upgrades to the running network.
+The important benefit of an HCU over a spork is that there is no downtime.
+Instead, at a protocol-determined height, the nodes of a specific role just reboot with a new software image and continue,
+while all other node roles remain available. Nevertheless, there are some notable disadvantages to HCUs in their current form:
+
+1. At the moment, HCUs are only supported for Execution Nodes.
+2. Time-sensitive coordination is necessary for node operators to make sure their node loads the correct image after the HCU but not too early
+   in case of an unforeseen crash. As software upgrades always entail some non-zero risk, all node operators need to be on standby during the HCU.
+3. When all Execution Nodes reboot synchronously, transaction processing stalls for a few minutes. Nevertheless, during Execution Nodes’ HCU,
+   the flow network continues to ingest and order transaction for subsequent execution.
```
Comment (jordanschalm) [inline README.md:125]:
```suggestion
   the Flow network continues to ingest and order transaction for subsequent execution.
```

**Comment by AlexHentschel** [comment]
> [Vishal] Please note that instead of individual git repo for each working group, there is now a single repo - https://github.com/onflow/workinggroups which houses all the working groups. Please can you update the readme.md within the subfolder for your working group instead of creating/updating a separate repo? I have copied over the latest from all individual repos to this comment repo.

This is is superseded by 
https://github.com/onflow/Flow-Working-Groups/pull/2

**Pull request closed without merging**